jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -403,6 +417,7 @@ def multipass_pipeline(jobs) {
                     SCRIPT_DIR,
                     server_era)
             step030_customize_settings(SCRIPT_DIR, job['puppet_settings'])
+            step035_customize_hocon_settings(SCRIPT_DIR, job['hocon_settings'])
```

Comment:
@cprice404 Do you think there's a chance that we'll want to use this to configure some settings that in PE could get wiped out / reset by the PE module during an agent run?  If so, we might want to move this step past the file_sync and classify_nodes steps like we did with `step080_customize_settings`.